sql/postgres/migrate.go

```diff
@@ -434,19 +435,31 @@ func (s *state) column(b *sqlx.Builder, c *schema.Column) {
 	b.P("NULL")
 	s.columnDefault(b, c)
 	for _, attr := range c.Attrs {
-		switch attr := attr.(type) {
+		switch a := attr.(type) {
 		case *schema.Comment:
 		case *schema.Collation:
-			b.P("COLLATE").Ident(attr.V)
+			b.P("COLLATE").Ident(a.V)
 		case *Identity:
-			if attr.Generation == "" {
-				attr.Generation = defaultIdentityGen
-			}
-			b.P("GENERATED", attr.Generation, "AS IDENTITY")
+			// Handled below.
```

Comment:
so either one attr is Identity and then we ignore it or the entire Attrs is an identity?

Reply:
We (see it and ) ignore it, because we don't accept unknown attributes. However, it's handled below with a helper function.